src/core/namespace.c

```diff
@@ -99,6 +99,7 @@ static const MountEntry protect_kernel_tunables_table[] = {
         { "/sys/kernel/debug",   READONLY,     true  },
         { "/sys/kernel/tracing", READONLY,     true  },
         { "/sys/fs/cgroup",      READWRITE,    false }, /* READONLY is set by ProtectControlGroups= option */
+        { "/sys/fs/selinux",     READWRITE,    true  },
```

Comment:
On the other hand, I think that many settings (including `Protect*`) should not be used on systems protected by `SELinux` because of https://github.com/systemd/systemd/issues/3845.